src/core/public/chrome/nav_links/nav_link.ts

```diff
@@ -65,6 +65,11 @@ export interface ChromeNavLink {
    */
   readonly href: string;
 
+  /**
+   * Path used by deepLinks to navigate, composed by the main app's `baseUrl` and the deepLink `path`.
+   */
+  readonly deepLinkPath?: string;
+
```

Comment:
We already have `baseUrl`, `url` and `href` in `ChromeNavLink`. Can't we reuse one of those here? I think we should be able to use `url` here, and then populate `url` accordingly?